### PR TITLE
Refactor Cognito Test

### PR DESCRIPTION
**Pull request by CedricProfessionnel**
Refactor cypress test for cognito

### CODE REVIEW

File: cypress/integration/aws-test/export-aws.spec.js [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
       cy.visit(`http://localhost:6001`)
-
       cy.wait(400)
       setLanguage()
       enterCredentialsCognitoS3()
       enterCredentialsUser()
       goToImportPage()
+    })
 
+    it("Try to export a natif project to aws", () => {
```
Comment (seveibar) [inline cypress/integration/aws-test/export-aws.spec.js:19]: *native?

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "proxy": "http://localhost:3000",
   "scripts": {
     "start": "cross-env PORT=6001 react-scripts start",
-    "start:desktop": "cross-env BROWSER=none USE_DEV_SERVER=yes concurrently 'npm run start' 'electron ./desktop'",
+    "start:desktop": "cross-env BROWSER=none USE_DEV_SERVER=yes concurrently 'start' 'electron ./desktop'",
```
Comment (seveibar) [inline package.json:10]: cool I didn't know this worked

**Review by seveibar — APPROVED** [review]
nice! looks good!

File: cypress/integration/aws-test/export-aws.spec.js (hunk 1 above)
Comment (CedricProfessionnel) [reply cypress/integration/aws-test/export-aws.spec.js:19]: An empty template

**Pull request merged**